### PR TITLE
Misc solid mechanics doco

### PR DESCRIPTION
**Pull request by GiudGiud**
This leaves the materials and UOs missing documentation

### CODE REVIEW

File: modules/solid_mechanics/doc/content/source/bcs/PresetVelocity.md [hunk 1]
```diff
@@ -1 +1,21 @@
-!template load file=stubs/moose_object.md.template name=PresetVelocity syntax=/BCs/PresetVelocity
\ No newline at end of file
+# PresetVelocity
+
+!syntax description /BCs/PresetVelocity
+
+This Dirichlet boundary conditions lets user set the displacement using the [!param](/BCs/PresetVelocity/function) and [!param](/BCs/PresetVelocity/velocity) parameters.
```
Comment (gambka) [inline modules/solid_mechanics/doc/content/source/bcs/PresetVelocity.md:5]:
```suggestion
This Dirichlet boundary condition lets the user set the displacement using the [!param](/BCs/PresetVelocity/function) and [!param](/BCs/PresetVelocity/velocity) parameters.
```

File: modules/solid_mechanics/doc/content/source/constraints/NodalFrictionalConstraint.md [hunk 2]
```diff
@@ -1 +1,41 @@
-!template load file=stubs/moose_object.md.template name=NodalFrictionalConstraint syntax=/Constraints/NodalFrictionalConstraint
+# NodalFrictionalConstraint
+
+!syntax description /Constraints/NodalFrictionalConstraint
+
+Only a penalty formulation is implemented.
+First the previous time step tangeantial force is computed as:
```
Comment (gambka) [inline modules/solid_mechanics/doc/content/source/constraints/NodalFrictionalConstraint.md:6]:
```suggestion
First. the previous time step tangential force is computed as:
```

File: modules/solid_mechanics/doc/content/source/constraints/NodalStickConstraint.md [hunk 3]
```diff
@@ -1 +1,23 @@
-!template load file=stubs/moose_object.md.template name=NodalStickConstraint syntax=/Constraints/NodalStickConstraint
+# NodalStickConstraint
+
+!syntax description /Constraints/NodalStickConstraint
+
+Both a kinematic and a penalty formulations are implemented.
```
Comment (gambka) [inline modules/solid_mechanics/doc/content/source/constraints/NodalStickConstraint.md:5]:
```suggestion
Both kinematic and penalty formulations are implemented.
```

File: modules/solid_mechanics/doc/content/source/postprocessors/Mass.md [hunk 4]
```diff
@@ -1 +1,13 @@
-!template load file=stubs/moose_object.md.template name=Mass syntax=/Postprocessors/Mass
\ No newline at end of file
+# Mass
+
+!syntax description /Postprocessors/Mass
+
+!alert note
+The `ADMass` postprocessor may be used with a density [AD](automatic_differentiation/index.md) material
```
Comment (gambka) [inline modules/solid_mechanics/doc/content/source/postprocessors/Mass.md:6]: Is this postprocessor only in `AD`? Is automatic differentiation required here?

Comment (GiudGiud) [reply modules/solid_mechanics/doc/content/source/postprocessors/Mass.md:6]: Mass will work with non-AD density
ADMass will work with AD density
there's no doco page for ADmass

File: modules/solid_mechanics/doc/content/source/constraints/NodalFrictionalConstraint.md [hunk 5]
```diff
@@ -3,7 +3,7 @@
 !syntax description /Constraints/NodalFrictionalConstraint
 
 Only a penalty formulation is implemented.
-First the previous time step tangeantial force is computed as:
+First. the previous time step tangential force is computed as:
```
Comment (gambka) [inline modules/solid_mechanics/doc/content/source/constraints/NodalFrictionalConstraint.md:6]: My original suggestion must have had a typo. I think this should be a comma, not a period after the word `First`

**Review by gambka — COMMENTED** [review]
Minor change and then this is ready to go.

**Review by gambka — APPROVED** [review]

**Review by lindsayad — APPROVED** [review]

**Pull request merged**